doc/user/quick-sql-grammar.md

```diff
@@ -228,6 +229,15 @@ Generate PL/SQL APIs on all tables for create, insert, update, delete and query.
 Adds an additional created, created_by, updated and updated_by columns to every
 table created.
 
+### boolean
+
+**Possible Values**: `yn`, `native`  
+**Default Value**: `false`
```

Comment:
It does not make sense that the possible values = yn/native, yet default value = false.

Please update accordingly. A default value has to equal one of the possible values.